Makefile

```diff
@@ -12,12 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REGISTRY_NAME=andyzhangx
+PKG=github.com/kubernetes-sigs/azurefile-csi-driver
+REGISTRY_NAME=ashishranjan738
```

Comment:
nit: don't use personal name for `REGISTRY_NAME`, but make it configurable via an environment variable.

Reply:
Oops, I forgot to change after changing thanks for pointing it out. Yeah taking it from env variable will be a good idea.

Reply:
will raise a seperate PR for this